### PR TITLE
remove 'extra log files' from config

### PR DESCRIPTION
**Pull request by wxtim**
These changes partially address #3696 
- [x] Appropriate tests are included (unit and/or functional).
- [x] No change log entry required: All changes to be logged at the end of issue #3696
- [x] Docs: All changes to be logged at the end of issue #3696

### CODE REVIEW

File: tests/flakyfunctional/events/01-task/flow.cylc [hunk 1]
```diff
@@ -25,7 +25,6 @@
 [runtime]
     [[root]]
         # make the event handler log available to gcylc:
-        extra log files = {{ EVNTLOG }}
```
Comment (MetRonnie) [inline tests/flakyfunctional/events/01-task/flow.cylc:28]: I think the comment on the line above should go as well?

**Review by MetRonnie — CHANGES_REQUESTED** [review]

**Review by kinow — APPROVED** [review]

File: tests/flakyfunctional/events/01-task/flow.cylc (hunk 1 above)
Comment (wxtim) [reply tests/flakyfunctional/events/01-task/flow.cylc:28]: Yep. Well Spotted. 🐆

File: tests/flakyfunctional/events/01-task/flow.cylc [hunk 2]
```diff
@@ -25,7 +25,6 @@
 [runtime]
     [[root]]
         # make the event handler log available to gcylc:
```
Comment (wxtim) [inline tests/flakyfunctional/events/01-task/flow.cylc:27]:
```suggestion
    [[root]]
```

Comment (wxtim) [inline tests/flakyfunctional/events/01-task/flow.cylc:27]:
```suggestion
    [[root]]
```

**Review by MetRonnie — APPROVED** [review]

**Pull request merged**